sdk/data/aztables/client.go

```diff
@@ -213,76 +220,57 @@ func newListEntitiesPage(resp *generated.TableClientQueryEntitiesResponse) (List
 
 // ListEntitiesPager is a Pager for Table entity query results.
 //
-// NextPage should be called first. It fetches the next available page of results from the service.
+// NextPage should be called first, it fetches the next available page of results from the service.
 // If the fetched page contains results, the return value is true, else false.
 // Results fetched from the service can be evaluated by calling PageResponse on this Pager.
 // If the result is false, the value of Err() will indicate if an error occurred.
 //
 // PageResponse returns the results from the page most recently fetched from the service.
-type ListEntitiesPager interface {
-	// PageResponse returns the current TableQueryResponseResponse.
-	PageResponse() ListEntitiesPage
-	// NextPage returns true if there is another page of data available, false if not
-	NextPage(context.Context) bool
-	// Err returns an error if there was an error on the last request
-	Err() error
-	// NextPagePartitionKey returns the PartitionKey for the current page
-	NextPagePartitionKey() *string
-	// NextPageRowKey returns the RowKey for the current page
-	NextPageRowKey() *string
+type ListEntitiesPager struct {
+	tableName   string
+	client      *generated.TableClient
+	current     generated.TableClientQueryEntitiesResponse
+	listOptions *ListEntitiesOptions
+	nextPK      *string
+	nextRK      *string
 }
 
-type tableEntityQueryResponsePager struct {
-	tableClient       *Client
-	current           *ListEntitiesPage
-	tableQueryOptions *generated.TableClientQueryEntitiesOptions
-	listOptions       *ListEntitiesOptions
-	err               error
+// NextPagePartitionKey returns one of the continuation tokens for the ListEntitiesPager.
+// Use in conjunction with the NewPageRowKey.
+func (p *ListEntitiesPager) NextPagePartitionKey() *string {
```

Comment:
Does this need to be pointer-to-string?  Same question for `NextPageRowKey()`

Reply:
When the pager is returned from `Client.List`, there are no continuation tokens. How should we return in this edge case? I tested with an empty string (`""`), but this will not page correctly. I think a string is the correct return type here, but when they are nil should I return an empty string?

Reply:
Looks like this has been resolved?